tests/model_fields/tests.py

```diff
@@ -661,6 +689,15 @@ def test_backend_range_validation(self):
             }
             with self.assertRaisesMessage(ValidationError, expected_message):
                 instance.full_clean()
+            try:
+                with transaction.atomic():
+                    instance.save()
+            except (DataError, IntegrityError):
+                pass  # backends such as Postgres don't fail silently
+            else:
+                qs = self.model.objects.filter(value__lte=min_value)
```

Comment:
My thinking is that now that we have the proper range, it's unlikely we'll create a regression for this on the stable branches so the second test isn't critical for the backport. Nonetheless, if you feel strongly about it, we can backport the feature flags too. I believe we need 1 to say whether or not a database raises an exception on out of range integer values (then we can use `assertRaises` instead of a silent pass) and another to say if the backend truncates integers to the highest value it can store. B the way, you mentioned SQLite has a similar behavior but `ops.integer_field_range` for that backend says "SQLite doesn't enforce any integer constraints" -- is it an error?